Makefile

```diff
@@ -1,7 +1,16 @@
+.PHONY = all testenv install package test test_in_containers test_in_container_37 test_in_container_38 clean container version
+
+HERE := $(shell pwd)
+HERE_OWNERSHIP := $(shell stat -c '%u:%g' $(HERE))
+FIX_OWNERSHIP := chown -R $(HERE_OWNERSHIP)
+
+SETUP_CONTAINER_FOR_TESTS := pip install tox; cd /src;pip install -r requirements.txt
```

Comment:
Why are we not installing `tox` via `requirements.txt`

Reply:
deh because I forgot to take that out after I got all of the tox dependencies pinned, removing now, thanks